radio/src/telemetry/frsky_sport.cpp

```diff
@@ -220,6 +226,19 @@ void sportProcessTelemetryPacket(const uint8_t * packet)
           sportProcessTelemetryPacket(id, 0, instance, data & 0xffff);
           sportProcessTelemetryPacket(id, 1, instance, data >> 16);
         }
+        else if (id >= ESC_TEMPERATURE_FIRST_ID && id <= ESC_TEMPERATURE_LAST_ID) {
+          sportProcessTelemetryPacket(id, 0, instance, data & 0x00ff);
+        }
+        else if (id >= GASSUIT_TEMP_FIRST_ID && id <= GASSUIT_TEMP_LAST_ID) {
```

Comment:
those 2 case can be combined (as I did for ESC), but perhaps the compiler does the optimization on his own